stripe/src/main/java/com/stripe/android/ApiRequest.java

```diff
@@ -89,9 +92,23 @@ static ApiRequest createDelete(@NonNull String url,
         if (options.stripeAccount != null) {
             headers.put("Stripe-Account", options.stripeAccount);
         }
+
+        if (SHOULD_INCLUDE_ACCEPT_LANGUAGE_HEADER) {
+            final String languageTag = getLanguageTag();
```

Comment:
do we usually `final` local variables?
I usually find that too verbose in java but no big deal, more of a style question :)

Reply:
I think it's generally good practice :)

Reply:
Converting this to Kotlin anyway :)